src/GraphQL.Tests/Types/FloatGraphTypeBadValueTests.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Threading.Tasks;
+using GraphQL.Types;
+using GraphQLParser.AST;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/pulls/1781
+    public class FloatGraphTypeBadValueTests : QueryTestBase<PR1781Schema>
+    {
+        [Fact]
+        public void BadFloatValues()
+        {
+            var type = new FloatGraphType();
+
+            var literal1 = $"{double.MaxValue:0}0.0";
+            var literal2 = $"{double.MinValue:0}0.0";
+
+            var value1 = new GraphQLFloatValue(literal1);
+            var value2 = new GraphQLFloatValue(literal2);
+
+            type.CanParseLiteral(value1).ShouldBeFalse();
+            type.CanParseLiteral(value2).ShouldBeFalse();
+
+            Should.Throw<InvalidOperationException>(() => type.ParseLiteral(value1)).Message.ShouldStartWith("Unable to convert '1797693134862320000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0' literal from AST representation to the scalar type 'Float'");
+            Should.Throw<InvalidOperationException>(() => type.ParseLiteral(value2)).Message.ShouldStartWith("Unable to convert '-1797693134862320000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0' literal from AST representation to the scalar type 'Float'");
+        }
+
+        [Fact]
+        public async Task DocumentExecuter_really_big_double_Invalid()
+        {
+            var de = new DocumentExecuter();
+            var number = $"{double.MaxValue:0}0.0";
+            number.ShouldBe("1797693134862320000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.0");
```

Comment:
@Shane32 From time to time, these two tests are completed with error even locally.

Reply:
It’s always the same tests and always because of the culture settings being changed in another test. The one test fails because the negative sign isn’t the correct character. The other test fails because the period isn’t the correct character. It fails often on my PC as well. I’m pretty sure if we disable the culture tests, the problems will go away - although I have not tested it.

Reply:
Yeah. I also saw difference in negative sign.